content/klc/S2.2.adoc

```diff
@@ -4,16 +4,12 @@ brief = "Non-functional variations in part number should be replaced with wildca
 
 Manufacturer Part Numbers (MPN) often include variations that are _non-functional_.
 
-Examples:
+Examples of parameters considered non-functional:
 
 * Temperature rating of the component
 * Packaging information (e.g. reel, tray, tape)
 * RoHS / PbFree information
 
-To capture every possible combination of part variation requires a large number of symbol aliases - this is to be avoided.
+To capture every possible combination of part variation would require a large number of symbol aliases - this is to be avoided.
```

Comment:
It would be more inline with the rest of KLC's wording to avoid a hyphen here and change '- this' to 'and'.

Reply:
done